pkg/skaffold/build/build.go

```diff
@@ -30,14 +30,20 @@ type Artifact struct {
 	Tag       string `json:"tag"`
 }
 
+type Result struct {
```

Comment:
please add godoc, explaining what the fields are too

Reply:
done